app/services/publish_manual_service.rb

```diff
@@ -38,6 +42,19 @@ def publish
     manual.publish
   end
 
+  def fetch_tags
+    @tags = TagFetcher.new(manual).fetch_tags.map { |t|
+      {
+        type: t.details.type,
+        slug: t.slug,
+      }
+    }
+  end
+
+  def update_manual_with_tags
+    manual.update({tags: @tags})
```

Comment:
It seems like we usually access instance variables in these classes via a `private attr_reader`